src/plugins/databases/index.js

```diff
@@ -783,6 +784,10 @@ export default class DatabasesPlugin extends FlipperPlugin<
             newFavorites = state.favorites.concat(value);
           }
         }
+        window.localStorage.setItem(
+          'favoritesQueries',
```

Comment:
It would be nice to have something a little more specific as `favoriteQueries` could apply to a variety of plugins we have. I don't know off the top of my head if we've got any better places for storing information that's supposed to persist across sessions, but I'll ask internally.